docs/developer-docs/ai/overview.mdx

```diff
@@ -4,32 +4,49 @@ keywords: [intermediate, concept, AI, ai, deAI, deai]
 
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
-# DeAI overview
+# Decentralized AI overview
 
 <MarkdownChipRow labels={["Intermediate", "Concept", "DeAI" ]} />
 
-## What is decentralized AI?
+Decentralized AI, or DeAI for short, is the moniker that describes the intersection between AI and blockchain technology.
+The term denotes a wide range of applications starting from peripheral elements such as tokenization or decentralized marketplaces and going all the way to running AI models fully onchain as smart contracts.
 
-Decentralized artificial intelligence (DeAI or decentralized AI) is a form of AI and machine learning that utilizes blockchain technologies and decentralized protocols such as ICP. ICP enables truly decentralized AI by running services on-chain, providing a permissionless and secure way to run machine learning applications. 
+## Why DeAI?
 
-Canisters can be used to host AI services and run machine learning algorithms using popular AI/ML frameworks and libraries. HTTPS outcalls can be used to make API calls to external AI APIs like OpenAI.
+DeAI in its truest form of running AI onchain as smart contracts can potentially solve the trust problem in AI.
+Today users have to blindly trust AI running on centralized servers.
+Users have no visibility into how their data is used and how AI models produce responses, whether they work correctly, reliably, and consistently.
+Since AI models behave like black boxes to users, building trustworthy AI models is a difficult challenge.
+This can be addressed if users are able to verify how a model has been trained and the inference process uses that very model that generates an output.
```

Comment:
```suggestion
DeAI in its truest form can potentially solve AI's trust problem. Today, users have to blindly trust AI running on centralized servers. They have no visibility into how their data is used, how AI models produce responses, or whether they work correctly, reliably, and consistently.

Since AI models behave like black boxes to users, building trustworthy AI models is a difficult challenge.
This can be addressed if users are able to verify how a model has been trained and that the inference process uses that very model to generate an output.

Trustworthy DeAI is possible on ICP using canister smart contracts.
```